src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java

```diff
@@ -199,36 +199,37 @@ public class ProGuardMojo extends AbstractMojo {
 	private boolean injarNotExistsSkip;
 
 	/**
-	 * Apply ProGuard classpathentry Filters to input jar. e.g. <code>!**.gif,!**&#47;tests&#47;**'</code>
+	 * Apply ProGuard classpathentry filters to input jar. e.g. <code>!**.gif,!**&#47;tests/**</code>
```

Comment:
Note: Cannot replace the other `&#47;` with `/` here because then Javadoc would interpret the `**/` as end of Javadoc comment.